packages/terra-modal/src/Modal.scss

```diff
@@ -1,25 +1,32 @@
 :local {
   .modal {
-    background-color: #fff;
-    border-radius: 5px;
+    background-color: var(--terra-modal-background-color, #fff);
+    border: var(--terra-modal-border, none);
```

Comment:
If there is no property value for default theme, lets drop the fallback value for these theming properties.

e.g. switch `border: var(--terra-modal-border, none);` to `border: var(--terra-modal-border);`

Looks like this can be done for the following

* border
* box-shadow
* color
* height
* margin
* max-width
* width

Reply:
https://github.com/cerner/terra-core/pull/1209/commits/5385d5a7b5cd88902bfd2a88ca54455ef554df0b